cli/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `addresses` now additionally prints the hex version of the address;
 - `outputs`, `unspent-outputs` print a list that includes number and type of the output;
 - `Account::switch` command to allow changing accounts quickly;
+- UX improvements (Ctrl+l, TAB completion/suggestions and more) during interactive account management;
```

Comment:
We already had TAB completion?

Reply:
yeah, but not exactly like this. Now ... if you TAB and there are multiple matches it will show a list. If you have no ideas whatsoever and TAB on an empty prompt it will show all available commands. Furthermore it will autocomplete commands from the history.